src/main/java/org/venice/beachfront/bfapi/model/oauth/GeoAxisCommonName.java

```diff
@@ -0,0 +1,67 @@
+package org.venice.beachfront.bfapi.model.oauth;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface GeoAxisCommonName {
+	public String toString();
+	
+	public static class Deserializer extends JsonDeserializer<GeoAxisCommonName> {
+		@Override
+		public GeoAxisCommonName deserialize(JsonParser p, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+			TreeNode node = p.getCodec().readTree(p);
+			if (node.isValueNode() && ((JsonNode)node).isTextual()) {
```

Comment:
I assume we shouldn't need to worry about any ClassCastExceptions here?  Not sure if it should always cast to JsonNode with no problems or if we need to add a catch just in case.

Reply:
It should always cast to JsonNode with no problems if `node.isValueNode()` is true.